packages/ember-simple-auth/tests/unit/session-stores/adaptive-test.js

```diff
@@ -1,98 +1,104 @@
 import { run } from '@ember/runloop';
-import {
-  describe,
-  beforeEach,
-  afterEach,
-  it
-} from 'mocha';
+import { describe, it } from 'mocha';
 import { expect } from 'chai';
 import sinonjs from 'sinon';
-import Adaptive from 'ember-simple-auth/session-stores/adaptive';
-import LocalStorage from 'ember-simple-auth/session-stores/local-storage';
 import itBehavesLikeAStore from './shared/store-behavior';
 import itBehavesLikeACookieStore from './shared/cookie-store-behavior';
 import FakeCookieService from '../../helpers/fake-cookie-service';
 import createAdaptiveStore from '../../helpers/create-adaptive-store';
+import { setupTest } from 'ember-mocha';
 
 describe('AdaptiveStore', () => {
-  let sinon;
-  let store;
-
-  beforeEach(function() {
-    sinon = sinonjs.createSandbox();
-  });
-
-  afterEach(function() {
-    store.clear();
-    sinon.restore();
-  });
-
+  setupTest();
   describe('when localStorage is available', function() {
-    beforeEach(function() {
-      store = Adaptive.extend({
-        _createStore(storeType, options) {
-          return LocalStorage.create({ _isFastBoot: false }, options);
-        }
-      }).create({
-        _isLocalStorageAvailable: true
-      });
-    });
-
     itBehavesLikeAStore({
-      store() {
+      store(sinon, owner) {
+        let store;
+        let cookieService;
+        owner.register('service:cookies', FakeCookieService);
+        cookieService = owner.lookup('service:cookies');
+        sinon.spy(cookieService, 'read');
+        sinon.spy(cookieService, 'write');
+        store = createAdaptiveStore(cookieService, {
+          _isLocal: false,
```

Comment:
This doesn't seem to exist anywhere actually? 🤔

Reply:
Yeah, that's true, I will remove it, also noticed that some of the `_isFastBoot` properties were miswritten, we have `_isFastBoot` and `_isFastboot` currently somewhere i believe.